src/helpers/prss.rs

```diff
@@ -36,6 +36,12 @@ pub struct PrssSpace {
     right: Generator,
 }
 
+impl PartialEq for PrssSpace {
+    fn eq(&self, other: &Self) -> bool {
+        self.generate_values(1) == other.generate_values(1)
```

Comment:
Hmmm... I guess this is one way to test it. Can't we compare the seeds instead? That seems better to me.

Reply:
That would have been ideal :D BUT we dont store the seed anywhere and Aes256 is not comparable. Hence, just compared the generated random values which returns (u128, u128) and chances of their being equal wrongly is very low.